### PR TITLE
P-6256: Added AI feature

### PR DESCRIPTION
**Pull request by elisebach**

### CODE REVIEW

**Review by mattvagni — APPROVED** [review]

File: plain-ai.mdx [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+title: 'Plain AI'
+---
+
+Quickly categorize and label new threads without lifting a finger. To turn on Plain’s AI features, go to **Settings → Workflow.\***
+
+<Frame>![Plain AI](/public/images/plain-ai.png)</Frame>
+
+**Auto-labelling**
+
+Plain AI will add 1-2 labels automatically to a thread based on its content. To best leverage Plain AI, make sure you’ve already set up [Labels](/labels) (**Settings → Labels**).
```
Comment (andrewblaney) [inline plain-ai.mdx:11]: Do we want to mention that labels won't be added to threads that already have labels (e.g. when a thread is created via the API with a label)

File: plain-ai.mdx [hunk 2]
```diff
@@ -0,0 +1,17 @@
+---
+title: 'Plain AI'
+---
+
+Quickly categorize and label new threads without lifting a finger. To turn on Plain’s AI features, go to **Settings → Workflow.\***
```
Comment (andrewblaney) [inline plain-ai.mdx:5]: not sure if this is a typo: 

```suggestion
Quickly categorize and label new threads without lifting a finger. To turn on Plain’s AI features, go to **Settings → Workflow.***
```

File: plain-ai.mdx (hunk 1 above)
Comment (ebach86) [reply plain-ai.mdx:11]: Yeah, that's probably a good idea. Do you have some good copy for that?

Comment (andrewblaney) [reply plain-ai.mdx:11]:
```suggestion
When a thread is created, Plain AI will add 1-2 labels automatically to a thread based on its content. If a thread was created with existing labels, no labels will be added. To best leverage Plain AI, make sure you’ve already set up [Labels](/labels) (**Settings → Labels**).
```

**Pull request merged**